lib/cmds/options.ts

```diff
@@ -37,6 +37,8 @@ export interface Server {
   version?: string;
   // Run as role = node option.
   runAsNode?: boolean;
+  // Run as grid node role = hub registration URL path.
+  gridNode?: string;
```

Comment:
Let's call this `gridUrl` or `nodeGridUrl`?

Reply:
ok - done